packages/shared/locales/en.json

```diff
@@ -908,6 +941,7 @@
         "reference": "Reference",
         "from": "From",
         "to": "To",
+        "or": "or",
```

Comment:
Instead of having `or` like that and consider the whitespaces similar to a latin format (I think some asian languages dont use whitespaces for this), I would rather do a "a or b" localization, something like
`"aOrB": "{optionA} or {optionB}`
(not loving the locale name `aOrB` though xD)